packages/google-cloud-translate/package.json

```diff
@@ -79,8 +94,7 @@
     "pack-n-play": "^1.0.0-2",
     "proxyquire": "^2.0.1",
```

Comment:
Can this be removed as well?

Reply:
I think we need it to run tests for CJS (since it doesn't support esmock)